modules/aws_account/main.tf

```diff
@@ -0,0 +1,178 @@
+terraform {
+  required_providers {
+    illumio-cloudsecure = {
+      source  = "illumio/illumio-cloudsecure"
+      version = "~> 1.0.11"
+    }
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.0"
+    }
+  }
+}
+
+data "aws_partition" "current" {}
+
+resource "random_uuid" "role_secret" {}
+
+resource "aws_iam_role_policy" "read" {
+  name = var.iam_name_prefix + "Policy"
+  tags = var.tags
+  role = aws_iam_role.role.id
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Effect   = "Allow"
+        Action   = [
```

Comment:
This is outdated and is missing the permissions added into the CloudFormation template on 2024-08-21.

Reply:
Fixed it.